apps/client/src/pages/LocationPage.js

```diff
@@ -60,7 +62,7 @@ const LocationPage = ({ topic }) => {
       if (focus) {
         document.activeElement.blur();
       } else {
-        history.push(geturl(routes.address, { slug }));
+        history.push(geturl(routes.address, topic));
```

Comment:
I think we need to be careful with this change, this need to be done everywhere

Reply:
Why? If `slug` is available from `topic`, then `{ slug }` is basically `{ slug: topic.slug }`. Topic already contains the slug property.